helm/values.yaml

```diff
@@ -80,8 +80,10 @@ autoscaling:
   targetCPUUtilizationPercentage: 40
   # targetMemoryUtilizationPercentage: 80
 
+# Pod disruption budget flag since PodDisruptionBudget doesn't take enabled
```

Comment:
Just describe the purpose of the flag, no need for reasoning/implementation details